datalad/resources/website/tests/test.html

```diff
@@ -91,6 +91,14 @@
 	  assert.deepEqual(click_handler({name: 'ds2', type: 'annex'}), {next: '?dir=/d1/ds2', type: 'search'}, 'url does not change if click current node');
 	  assert.deepEqual(click_handler({name: 'd2', type: 'dir'}), {next: '?dir=/d1/d2', type: 'search'}, 'url does not change if click current node');
       });
+      QUnit.test("Size Rendering", function(assert) {
+	assert.equal(size_renderer({ondisk: "0 Bytes", total: "1 Byte"}), "-/1 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({ondisk: "1 Byte", total: "1 Byte"}), "1 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({ondisk: "1 Byte", total: "2 Byte"}), "1 Byte/2 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({total: "1 Byte"}), "-/1 Byte", "ondisk size equals 0");
+	assert.equal(size_renderer({}), "-/-", "ondisk size equals 0");
```

Comment:
yeah just pushed a commit to fix those 'ondisk size equals 0'. Those are test comments, to display what's being tested in test.html UI/code.

The metadata json contains human sizes only. The machine to human size conversion is being currently done in the python metadata generator(`ls --json`). I should do that in js though.

Need to look into tooltips on columns but shouldn't be too painful. [This](https://datatables.net/beta/1.9/examples/advanced_init/events_post_init.html) generates tooltip for each row